config/routes.rb

```diff
@@ -2,8 +2,11 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "users#index"
+   # root "articles#index" 
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
Noted. Thanks